README.md

```diff
@@ -54,6 +54,23 @@ For these tasks you may prefer using webpki in combination with libraries like
 Changelog
 =========
 
+* Next release:
+  - *Breaking change*: added `EndEntity::verify_is_valid_tls_client_cert`
+    argument for certificate revocation lists.
+  - Added certificate revocation list (CRL) support.
```

Comment:
I feel like we should either have this line first (as the headline feature of the release) or keep all the breaking changes first (as the thing most likely to impact downstream users), but avoid keeping this in between the breaking changes.

Reply:
I re-arranged the items a little bit. Previously I wrote them more or less in the order they landed in-tree. Now they're written with breaking changes first, then the CRL work, then small misc changes, and the MSRV update last.